stores/metadata.go

```diff
@@ -124,6 +124,7 @@ type (
 	dbSlice struct {
```

Comment:
Reminder to test and merge https://github.com/SiaFoundation/renterd/pull/747 first